LayoutKitSampleApp/Benchmarks/CollectionViewController.swift

```diff
@@ -8,6 +8,23 @@
 
 import UIKit
 
+
+/// Each view type of collection view should subclass from `CollectionViewController`
+/// If each view type uses `CollectionViewController`, it will cause the crash.
+/// The crash reproduce steps: BenchmarkViewController -> UICollectionViewUIStackFeed
+/// -> Back -> Any other UICollectionView
+/// The reason behinds this is `DWURecyclingAlert` injects benchmark and label by using
+/// method swizzling. Method swizzling doesn't work well with generics. The second time
+/// of method sizzling would not work and bring into infinite loop if different view type
+/// uses `CollectionViewController` directly.
+/// We also need to revisit this if `DWUReclingAlert` should be still used.
```

Comment:
Here is my thought, instead of using `DWURecyclingAlert`, we may take the label and benchmark part and implement method swizzling by ourselves. That will avoid some unnecessary incorrect method swizzling.

Alternatively, we may look other solutions, like other libraries, or we can inject label manually.